runtime-light/stdlib/stdlib.cmake

```diff
@@ -1,18 +1,21 @@
 prepend(
-        RUNTIME_STDLIB_SRC
-        stdlib/
-        component/component-api.cpp
-        curl/curl.cpp
-        exit/exit-functions.cpp
-        fork/fork-context.cpp
-        output/output-buffer.cpp
-        output/print-functions.cpp
-        rpc/rpc-api.cpp
-        rpc/rpc-context.cpp
-        rpc/rpc-extra-headers.cpp
-        rpc/rpc-extra-info.cpp
-        rpc/rpc-tl-error.cpp
-        rpc/rpc-tl-query.cpp
-        rpc/rpc-tl-request.cpp
-        string/concat.cpp
-        regex/regex-functions.cpp)
+  RUNTIME_STDLIB_SRC
```

Comment:
Why you format this file? Do you use IDE or what? My IDE format this file in different way. Let's come to a common decision

Reply:
It's a default formatting in my editor. Let's do this :)

Reply:
Okay
Can you format modified files in the same way?

Reply:
They already are.